.github/workflows/presubmit.yml

```diff
@@ -72,26 +71,23 @@ jobs:
         run: |
           cd room && ./gradlew :buildOnServer && cd ../work && ./gradlew :buildOnServer
         id: gradlew_bos
-      - name: Set webhook outputs from ./gradlew
-        if: always()
-        run: |
-          if [ "${{ steps.gradlew_bos.outcome }}" == "success" ] || [ "${{ steps.gradlew_bos.outcome }}" == "skipped" ]
-          then
-            echo "##[set-output name=success;]true"
-          else
-            echo "##[set-output name=success;]false"
-          fi
-        id: gradlew_bos_outputs
       - name: Upload Results
         if: always()
         uses: actions/upload-artifact@v2
         with:
           name: artifacts
           path: ~/dist
-      - name: Completed WebHook
-        if: always()
+      - name: Successful WebHook
+        if: ${{ success() }}
+        uses: tikurahul/github-workflow-webhook-action@master
+        with:
+          url: 'https://androidx.dev/github/androidX/presubmit/hook'
+          secret: ${{ secrets.ANDROIDX_PRESUBMIT_HOOK_SECRET }}
+          payload: '{ "platform": "${{ matrix.os }}", "token": "${{ secrets.GITHUB_TOKEN }}", "state": "completed", "success": true }'
```

Comment:
it kinda sucks that we can't inline the if failure / success check here? That looks like the only difference to me from these two branches

Reply:
Yes, it does. But unfortunately there is no easy way to inline this without having to write more actions. :disappointed: